.travis.yml

```diff
@@ -1,29 +1,23 @@
 language: c
 env:
+###############################################################################
   # FIXME: If we want to check assembly too SYNTAX_ONLY needs to be 0
   # FIXME: We'd like to build with WARNINGS_AS_ERRORS=1 but there are too
   # many warnings firing right now
   # Note: Can't use CC variable as TravisCI will override it
-  - C_COMPILER=gcc-4.8 SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
+  - C_COMPILER=gcc SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
```

Comment:
I don't want two scripts because they would end up doing duplicate work which leads to more maintenance effort. Instead I've moved most of the environment variable declarations out of ``.travis.yml`` and into ``scripts/travisci-build.sh``.